### DISCUSSION TITLE
Barrier and screen wake

### DISCUSSION THREAD

**Discussion by snoopiedoom** [debauchee/barrier#1889] [Ideas]
Heres my question, could Barrier be prevented from waking the screen ? Working from home, I've setup the Barrier server on a linux workstation and client on Windows workstation, and it looks really good when only Windows screens are turned on. Sadly, the moment I start using the keyboard and mice on client, it wakes the server machine. ☹️ 

So heres my idea, would it be possible to have Barrier wake the screen only on the machines being interacted with ? In that case, I'd be able to wake specific machines even.

I'd happily contribute, but I've yet to make any progress. (still stuck [here](https://unix.stackexchange.com/questions/596894/how-does-linuxs-display-work#:~:text=65-,Linux%20display,-The%20Linux%27s%20display)).

**Comment by NaturalCoder** [comment]
Using linux on both server and client work that way..., I do have another problem though, when I leave the mouse on the client screen, It stops the server screen safer... Very annoying because every time I forget to bring the mouse back my monitors stays on indefinitely
